### DISCUSSION TITLE
Progress bar design

### DISCUSSION THREAD

**Discussion by ySergi** [lobehub/sd-webui-lobe-theme#562] [Ideas]
![image](https://github.com/lobehub/sd-webui-lobe-theme/assets/138529583/cd7a8def-3465-49f9-bc48-03befad60983)
I think the image is self-explanatory... looks horrible af, and needs to be re-designed. It doesn't fit with the theme's aesthetics.

**Comment by realmalikov** [comment]
2 years later, nothing has changed haha. You are absolutely right, this progress bar is absolutely unworthy of the otherwise beautiful UI.
